ports/gecode/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "gecode",
+  "version-string": "6.2.0",
+  "description": "Gecode is an open source C++ toolkit for developing constraint-based systems and applications. Gecode provides a constraint solver with state-of-the-art performance while being modular and extensible.",
+  "homepage": "https://github.com/Gecode/gecode"
```

```suggestion
  "homepage": "https://github.com/Gecode/gecode",
  "supports": "linux | osx"
```
Since currently, it doesn't support Windows, we can add `supports `here to make sure all CI checks passed.